static/index.html

```diff
@@ -1,129 +1,160 @@
+    <script src="leaflet/leaflet.rotatedMarker.js"></script>
+    <script src="leaflet/leaflet.svgIcon.js"></script>
+    <script src="leaflet/leaflet.pattern.js"></script>
+    <link rel="stylesheet" href="leaflet/L.Icon.Pulse.css" />
+    <script src="leaflet/L.Icon.Pulse.js"></script>
+    <link rel="stylesheet" href="leaflet/L.Control.Zoominfo.css" />
+    <script src="leaflet/L.Control.Zoominfo.js"></script>
+    <link rel="stylesheet" href="leaflet/L.Control.Basemaps.css" />
+    <script src="leaflet/L.Control.Basemaps-min.js"></script>
+    <script src="scripts/ocap.marker.js"></script>
+    <script src="scripts/ocap.entity.js"></script>
+    <script src="scripts/ocap.event.js"></script>
+    <script src="scripts/ocap.group.js"></script>
+    <script src="scripts/ocap.groups.js"></script>
+    <script src="scripts/ocap.vehicle.js"></script>
+    <script src="scripts/ocap.unit.js"></script>
+    <script src="scripts/ocap.ui.js"></script>
+    <script src="scripts/ocap.js"></script>
+  </head>
+  <body>
+    <div id="container">
+      <div id="map"></div>
+      <div id="topPanel">
+        <div id="shareButton" class="bold button"></div>
+        <div id="aboutButton" class="bold button">i</div>
+        <div id="statsButton" class="bold button hiddenExperimental">S</div>
+        <span id="missionName" class="medium"></span>
+      </div>
+      <div id="leftPanel">
+        <div class="title bold" data-lb="players"></div>
+        <div class="panelContent">
+          <ul id="listWest"></ul>
+          <ul id="listEast"></ul>
+          <ul id="listGuer"></ul>
+          <ul id="listCiv"></ul>
+        </div>
+        <div id="controlSide">
+          <div id="sideWest" class="blufor sideTitle">BLUFOR</div>
+          <div id="sideEast" class="opfor sideTitle">OPFOR</div>
+          <div id="sideGuer" class="ind sideTitle">IND</div>
+          <div id="sideCiv" class="civ sideTitle">CIV</div>
+        </div>
+      </div>
+      <div id="rightPanel">
+        <div class="title bold" data-lb="events"></div>
+        <div class="filterBox">
+          <div id="filterHitEventsButton" class="filterHit"></div>
+          <div id="filterConnectEventsButton" class="filterConnect"></div>
+          <input
+            type="text"
+            id="filterEventsInput"
+            placeholder=""
+            data-lb="filter"
+          />
+        </div>
+        <div class="panelContent">
+          <ul id="eventList"></ul>
+        </div>
+      </div>
+      <div class="extraInfoBox">
+        <div class="extraInfoBoxContent">
+          <span class="bold">Cursor target: </span
+          ><span id="cursorTargetBox">None</span>
+        </div>
+      </div>
+      <div id="bottomPanel">
+        <div id="frameSliderContainer">
+          <div id="eventTimeline"></div>
+          <div class="frameSliderContainer2">
+            <input type="range" id="frameSlider" min="0" value="0" />
```

Comment:
In html a / at the end of an element is not needed, its only part of the xhtml specification